visualizer/parseTree.js

```diff
@@ -134,7 +144,23 @@
       if (!el._input) {
         continue;
       }
-      el._input.style.minWidth = el.offsetWidth + 'px';
+
+      if (!el.classList.contains('hidden') &&
+          domUtil.closestElementMatching('.collapsed', el) == null) {
+        var elStyle = getComputedStyle(el);
```

Comment:
Nit: I'd drop the `el` from the beginning of all these names...I don't think it improves the clarity.